site/docs/get-started/quickstart.md

```diff
@@ -108,28 +108,76 @@ Run the following to install vault and the plugin in development mode:
     kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=120s
     ```
 
-## Install control plane
+## Install data catalog
 
-??? tip "Install latest development version from GitHub"
+Fybrik requires a data catalog in which to store metadata related to data assets. Currently, two data catalogs are supported:
```

Comment:
Changed the wording.
I remove most mentions of `katalog`. For now, I keep `katalog` in the list of supported data catalog, since it is still needed for the `module chaining` scenario. I moved it to second place.